.bldr.toml

```diff
@@ -1910,6 +1922,30 @@ paths = [
   "vendor/google.golang.org/grpc/*",
   "vendor/gopkg.in/yaml.v2/*"
 ]
+
+[chef/automate-minio]
```

Comment:
`automate-builder-minio`? Or are we using the same minio for everything?

Reply:
not sure yet. its one of the areas that needs to be thought out